### PR TITLE
mysql2

### PR DESCRIPTION
**Pull request by Qard**
Fixes #211.

### CODE REVIEW

File: lib/instrumentation/modules/mysql2.js [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+      }
+
+      return result
+
+      function wrapCallback (cb) {
+        hasCallback = true
+        if (span) span.start()
+        return agent._instrumentation.bindFunction(span ? wrappedCallback : cb)
+        function wrappedCallback () {
+          if (span) span.end()
```
Comment (watson) [inline lib/instrumentation/modules/mysql2.js]: `wrappedCallback` will only ever get called if there is a span

File: lib/instrumentation/modules/mysql2.js [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+        arguments[1] = wrapCallback(values)
+      } else if (typeof cb === 'function') {
+        arguments[2] = wrapCallback(cb)
+      }
+
+      var result = original.apply(this, arguments)
+      if (result && !hasCallback) {
+        ins.bindEmitter(result)
+        if (span) {
+          let started = false
```
Comment (watson) [inline lib/instrumentation/modules/mysql2.js]: Nitpick: we use `var` everywhere else in this file, so it seems odd to suddenly use `let` here (not a merge blocker, just an observation)

Comment (Qard) [reply lib/instrumentation/modules/mysql2.js]: changed to var

File: lib/instrumentation/modules/mysql2.js (hunk 1 above)
Comment (Qard) [reply lib/instrumentation/modules/mysql2.js]: fixed

**Comment by Qard** [comment]
Updated the Node.js 10 section of the `.travis.yml` TAV settings.

**Pull request merged**